Uplift/Controllers/HomeViewController.swift

```diff
@@ -40,21 +41,25 @@ class HomeViewController: UIViewController {
         static let lookingForListCellIdentifier = "lookingForListCellIdentifier"
         static let todaysClassesListCellIdentifier = "todaysClassesListCellIdentifier"
         static let todaysClassesEmptyCellIdentifier = "todaysClassesEmptyCellIdentifier"
+        static let sportsListCellIdentifier = "sportsListCellIdentifier"
     }
 
     // MARK: - Enums
     enum SectionType: String {
         case checkIns = "DAILY CHECK-INS"
         case myGyms = "MY GYMS"
         case todaysClasses = "TODAY'S CLASSES"
+        case yourSports = "Your Sports"
         case lookingFor = "I'M LOOKING FOR..."
     }
 
     override func viewDidLoad() {
         super.viewDidLoad()
 
         //MARK: removed .lookingFor
-        sections = [.myGyms, .todaysClasses]
+        sections = [.todaysClasses, .yourSports, .myGyms]
+
+        sports = [Sport(name:"Lifting", image: UIImage(named: "lifting")!), Sport(name:"Basketball", image: UIImage(named: "basketball1")!), Sport(name:"Bowling", image: UIImage(named: "bowling1")!), Sport(name:"Swimming", image: UIImage(named: "swimming")!), Sport(name:"Lifting", image: UIImage(named: "lifting")!), Sport(name:"Basketball", image: UIImage(named: "basketball1")!)]
```

Comment:
Can you reformat this so each Sport is on a new line, it would look a lot more readable

Reply:
Also you may want to put the images in a constants file